AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -342,6 +342,7 @@ peacocktv.com##.playback-ad-blocker-notice
 psychic.de#%#//scriptlet("prevent-setTimeout", "test.offsetHeight")
 esoligorsk.by###stopAdBlock
 wikihow.com###contribute_footer
+infranken.de###wl-container
```

Comment:
Better to add domain to existing rule (English filter -> content_blocker.txt too):

#@#.banner_ad
#@#.full_ad